lm_eval/tasks/fld/multiple_choice_question/_base.yaml

```diff
@@ -0,0 +1,14 @@
+dataset_path: hitachi-nlp/FLD.v2
+dataset_name: default
+output_type: multiple_choice
+training_split: train
+validation_split: validation
+test_split: test
+doc_to_target: world_assump_label
+doc_to_choice: ["PROVED", "DISPROVED", "UNKNOWN"]
```

Comment:
in your testing, do you find that models perform well with these as the choices (without few-shot examples?)

Also, renaming `_base.yaml` -> `_base_yaml` for both these template yamls will cause the template stubs to not erroneously register as tasks!